.travis.yml

```diff
@@ -27,19 +27,16 @@ before_install:
     - npm install -g grunt-cli
 
 install:
-    - composer update --prefer-dist --no-interaction
     - mysql -u root -e 'CREATE DATABASE akeneo_pim;'
     - mysql -u root -e 'CREATE USER "akeneo_pim"@"localhost" IDENTIFIED BY "akeneo_pim";'
     - mysql -u root -e 'GRANT ALL ON akeneo_pim.* TO "akeneo_pim"@"localhost";'
     - composer update --prefer-dist --no-interaction
-    - ./app/console pim:install
-    - ./app/console oro:requirejs:generate-config
+    - cp ./app/config/parameters_test.yml.dist ./app/config/parameters_test.yml
+    - ./app/console --env=test pim:install
+    - ./app/console --env=test oro:requirejs:generate-config
```

Comment:
Sorry, not sure I understand your question. You mean "the link with your PR"?
Just that Travis cannot run as we install icecat instead of minimal. As icecat contains products, and we break a lot of stuff for the moment, you can't install a catalog with products. But minimal is fine.

What's more, it is better for Travis performances, so I will do the same on master when I'll upgrade to last Symfony 2.7 patch.